Scripts/Modules/HighlightModule/HighlightModule.cs

```diff
@@ -28,19 +28,24 @@ sealed class HighlightModule : MonoBehaviour, IUsesGameObjectLocking
 		}
 		readonly List<Func<GameObject, Material, bool>> m_CustomHighlightFuncs = new List<Func<GameObject, Material, bool>>();
 
-		public Color leftColor
-		{
-			get { return m_LeftHighlightMaterial.color; }
-		}
-
-		public Color rightColor
+		public Color highlightColor
 		{
-			get { return m_RightHighlightMaterial.color; }
+			get { return m_RayHighlightMaterial.GetVector("_Color"); }
+			set { m_RayHighlightMaterial.color = value; }
 		}
 
 		void Awake()
 		{
 			s_BakedMesh = new Mesh();
+
+			if (EditorPrefs.HasKey(k_SelectionOutlinePrefsKey))
+			{
+				var selectionColor = MaterialUtils.PrefToColor(EditorPrefs.GetString(k_SelectionOutlinePrefsKey));
+				selectionColor.a = 1;
+
+				m_RayHighlightMaterial = Instantiate(m_RayHighlightMaterial);
```

Comment:
I think we should always instantiate - otherwise this material is inconsistently what the field has referenced or an instance of it